pylint/extensions/comparison_placement.py

```diff
@@ -0,0 +1,64 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+"""Checks the placement of comparisons"""
```

Comment:
```suggestion
"""
Checks for yoda comparisons (variable before constant)
See https://en.wikipedia.org/wiki/Yoda_conditions
"""
```
I think it's ok to be explicit and use colloquialism